hail/src/main/scala/is/hail/io/vcf/LoadVCF.scala

```diff
@@ -1320,3 +1707,15 @@ class VCFsReader(
     }
   }
 }
+
+class BufferedLineIterator(bit: BufferedIterator[String]) extends htsjdk.tribble.readers.LineIterator {
```

Comment:
Can you move this to the top of the file?

Reply:
sure